### ISSUE TITLE
Event data is wonky

### ISSUE THREAD

**Issue by mikebell** [nwdrupal/nwdrupalwebsite#8]
![up-coming events north west drupal 2016-01-18 16-07-22](https://cloud.githubusercontent.com/assets/242104/12396168/ef483d9c-bdfd-11e5-998c-83ed8cebade8.png)

* All the dates are the same
* Some have venue (might be data source issue)
* Can't RSVP to some events

**Comment by mikebell** [comment]
content.field_event_date.value doesn't exist in the twig object. I'm not sure where to correctly pull this value from.

It seems like the theme layer in D8 has even more arrays and they're even more complex than before.

**Comment by philipnorton42** [comment]
So I've learnt a bit about how the date field works in doing this (which was partly the intention of doing this project). Rather than try to do anything with the date field within the template I have extracted the information in a preprocess step and passed that to the template.

TL;DR : The event date times are no longer wonky.
